README.md

```diff
@@ -10,8 +11,10 @@ Syntax highlighting and indentation for [Purescript][] based on [idris-vim][] an
 I recommend using [Pathogen][] for installation. Simply clone
 this repo into your `~/.vim/bundle` directory and you are ready to go.
 
-    cd ~/.vim/bundle
-    git clone https://github.com/raichoo/purescript-vim.git
+```sh
```

Comment:
This is actually a shell session (use `console`, `sh-session`, or `shell-session`) and not a Bash script.